src/analysis/convex/cone.lean

```diff
@@ -36,13 +36,29 @@ We prove two extension theorems:
   then `f` can be extended to the whole space to a linear map `g` such that `g x ≤ N x`
   for all `x`
 
+We prove the following theorems:
+* `hyperplane_separation_of_nonempty_of_is_closed_of_nmem`:
```

Comment:
The hyperlinks in our docs won't work if you don't give the full name:
```suggestion
* `convex_cone.hyperplane_separation_of_nonempty_of_is_closed_of_nmem`:
```

Reply:
I see, thanks!